[ffmpeg[openssl]] build failure on Windows

**Host Environment**
- OS: Windows 10
- Compiler: Visual Studio 2017 Enterprise

**To Reproduce**
Steps to reproduce the behavior:
`vcpkg install ffmpeg[openssl]:x64-windows`

**Failure logs** 
```
-- Building ffmpeg for Release
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:72 (message):
    Command failed: D:/vcpkg/downloads/tools/msys2/msys64/usr/bin/bash.exe --noprofile --norc D:/vcpkg/ports/ffmpeg\build.sh D:/vcpkg/buildtrees/ffmpeg/x64-windows-rel D:/vcpkg/buildtrees/ffmpeg/src/n4.2-02d8c63f80 D:/vcpkg/packages/ffmpeg_x64-windows "--enable-asm --enable-yasm --disable-doc --enable-debug --enable-runtime-cpudetect --enable-nonfree --enable-openssl --disable-ffmpeg --disable-ffplay --disable-ffprobe --disable-libvpx --disable-libx264 --disable-opencl --disable-lzma --disable-bzlib --enable-avresample  --disable-static --enable-shared --extra-cflags=-DHAVE_UNISTD_H=0  --extra-cflags=-MD --extra-cxxflags=-MD"
    Working Directory: D:/vcpkg/buildtrees/ffmpeg/x64-windows-rel
    Error code: 1
    See logs for more information:
      D:\vcpkg\buildtrees\ffmpeg\build-x64-windows-rel-out.log
...
D:\vcpkg>type D:\vcpkg\buildtrees\ffmpeg\build-x64-windows-rel-out.log
=== CONFIGURING ===
ERROR: openssl not found
...
```

**Additional context**
This started failing in the last day or two.  All other ffmpeg features still build fine with x64-windows.

--- a/ports/ffmpeg/0002-detect-openssl.patch
+++ b/ports/ffmpeg/0002-detect-openssl.patch
@@ -7,7 +7,7 @@ index 1c8008a..bd2de34 100755
                                 check_lib openssl openssl/ssl.h SSL_library_init -lssl32 -leay32 ||
                                 check_lib openssl openssl/ssl.h SSL_library_init -lssl -lcrypto -lws2_32 -lgdi32 ||
 -                               die "ERROR: openssl not found"; }
-+                               check_lib openssl openssl/ssl.h SSL_library_init -llibeay32 -lssleay32 ||
++                               check_lib openssl openssl/ssl.h OPENSSL_init_ssl -llibssl -llibcrypto -lws2_32 -lgdi32 -lcrypt32 ||
 +                               die "ERROR: openssl not found"; }
  enabled pocketsphinx      && require_pkg_config pocketsphinx pocketsphinx pocketsphinx/pocketsphinx.h ps_init
  enabled rkmpp             && { require_pkg_config rkmpp rockchip_mpp  rockchip/rk_mpi.h mpp_create &&

--- a/ports/ffmpeg/CONTROL
+++ b/ports/ffmpeg/CONTROL
@@ -1,5 +1,5 @@
 Source: ffmpeg
-Version: 4.2-2
+Version: 4.2-3
 Build-Depends: zlib
 Homepage: https://ffmpeg.org
 Description: a library to decode, encode, transcode, mux, demux, stream, filter and play pretty much anything that humans and machines have created.

